recipes/ocl-icd-system/activate.sh

```diff
@@ -0,0 +1,12 @@
+conda_ocl_icd_system_activate () {
+    # If no OpenCL ICD has been installed with conda yet
+    if [[ -z "$(ls ${CONDA_PREFIX}/etc/OpenCL/vendors/)" ]]; then
+        # and a system installation is available
+        if [[ ! -z "$(ls /etc/OpenCL/vendors/*.icd 2> /dev/null || true)" ]]; then
+            ln -s /etc/OpenCL/vendors "${CONDA_PREFIX}"/etc/OpenCL/vendors/system
```

Comment:
I would create this symlink irrespective of whether there's a system installation or not.